development/docker-compose.yml

```diff
@@ -109,8 +109,6 @@ services:
         condition: service_healthy
       message-queue:
         condition: service_healthy
-      cache:
```

Comment:
I think we need this one because this file is use not use during the tests, it's used to build the demo and the dev environment